docs/panel.md

```diff
@@ -0,0 +1,129 @@
+# Panel widget
+
+A composable [panel](https://bulma.io/documentation/components/panel/), for compact controls
+
+## Usage
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Yii\Bulma\Panel;
+use Yiisoft\Yii\Bulma\Asset\BulmaAsset;
+use Yiisoft\Yii\Bulma\Asset\BulmaJsAsset;
+
+/**
+ * @var \Yiisoft\Assets\AssetManager $assetManager
+ * @var \Yiisoft\View\WebView $this
+ */
+
+$assetManager->register([
+    BulmaAsset::class,
+    BulmaJsAsset::class,
+]);
+
+$this->setCssFiles($assetManager->getCssFiles());
+$this->setJsFiles($assetManager->getJsFiles());
+
+$template =<<<HTML
```

Comment:
Is it necessary to provide template every time when using the widget? If so, that's very inconvenient and I'd better do it with simple PHP and HTML...

Reply:
> Is it necessary to provide template every time when using the widget?

No, there is a default template https://github.com/yiisoft/yii-bulma/blob/c9e5db7240b5518a5de973a94bf83dee8f7cbd9f/src/Panel.php#L35

Reply:
Would it generate proper bulma markup?

Reply:
Yes. https://github.com/yiisoft/yii-bulma/blob/214ffc7cecb6addbf21d8142c87b82d330b76d83/tests/PanelTest.php#L188